Jenkinsfile

```diff
@@ -192,12 +195,20 @@ pipeline {
                             junit 'nosetests-centos.xml'
                             archiveArtifacts artifacts: '**/*.log'
                         }
+                        cleanup {
+                        dir('test/output') {
```

Comment:
Is GitHub just not displaying the indentation properly here?

Reply:
Probably tabs vs spaces, I did try to correct it but this file is all over the place anyway.